src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 900;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.searchBar-1 {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+  display: block;
+}
+
+.search-1 {
+  width: 100%;
+  position: relative;
+  padding-left: 62px;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
```

Comment:
Move all repeatable colors to CSS variables. CSS variables must be in the `:root` selector.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
I did it, HOWEVER, I don't think that moving color that repeats twice is a good idea, for me it is uncomfortable + it makes the length of the code line more than 80 symbols